Pod/Classes/PINAlternateRepresentationDelegate.h

```diff
@@ -0,0 +1,25 @@
+//
+//  PINAlternateRepresentationDelegate.h
+//  Pods
+//
+//  Created by Garrett Moon on 3/17/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import "PINRemoteImageManager.h"
+
+@protocol PINRemoteImageManagerAlternateRepresentationDelegate <NSObject>
```

Comment:
It seems the only method here is providing an alternate representation, so I wonder if it would make sense to call this protocol `PINRemoteImageManagerAlternateRepresentationProvider` or something like it. Thoughts?